packages/css/src/components/Modal/modal.css

```diff
@@ -0,0 +1,52 @@
+.modal {
+    position: absolute;
+    max-width: 660px;
```

Comment:
Why did you pick 660px - isn't there a predefined value?

How easy is for people to override this

It would probably be best if this comes as a design system variable, let's bring it up in the meeting today

Reply:
Agree, we should check that in the meeting today

Reply:
@KerimRahman and @kidroca—Created https://github.com/compassion-global-experience/cds-tokens/issues/20 for this kind of sizing issue.